packages/block-library/src/social-link/index.php

```diff
@@ -32,8 +32,10 @@ function render_block_core_social_link( $attributes, $content, $block ) {
 		$attribute = 'rel="noopener nofollow" target="_blank"';
 	}
 
-	$icon = block_core_social_link_get_icon( $service );
-	return '<li class="wp-social-link wp-social-link-' . esc_attr( $service ) . esc_attr( $class_name ) . '"><a href="' . esc_url( $url ) . '" aria-label="' . esc_attr( $label ) . '" ' . $attribute . '> ' . $icon . '</a></li>';
+	$icon               = block_core_social_link_get_icon( $service );
+	$wrapper_attributes = get_block_wrapper_attributes( array( 'class' => 'wp-social-link wp-social-link-' . $service . $class_name ) );
+
+	return '<li .' . $wrapper_attributes . '><a href="' . esc_url( $url ) . '" aria-label="' . esc_attr( $label ) . '" ' . $attribute . '> ' . $icon . '</a></li>';
```

Comment:
There's a typo here, note the dot within the string `<li .`.

Reply:
Fix at https://github.com/WordPress/gutenberg/pull/26282

Reply:
Good catch, thanks for the PR